packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerPopup.stories.tsx

```diff
@@ -1,6 +1,15 @@
 import * as React from 'react';
-import { makeStyles, shorthands, Button, Popover, PopoverSurface, PopoverTrigger } from '@fluentui/react-components';
+import {
+  makeStyles,
+  shorthands,
+  Button,
+  Popover,
+  PopoverSurface,
+  useRestoreFocusTarget,
+  useId,
+} from '@fluentui/react-components';
 import { SwatchPicker, SwatchPickerOnSelectEventHandler, ColorSwatch } from '@fluentui/react-swatch-picker-preview';
```

Comment:
Sorry, what I meant was from the `@fluentui/react-swatch-picker-preview`.

```suggestion
import { SwatchPicker, ColorSwatch } from '@fluentui/react-swatch-picker-preview';
import type { SwatchPickerOnSelectEventHandler } from '@fluentui/react-swatch-picker-preview';
```

Reply:
fixed :)